src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -369,6 +369,36 @@ public boolean isCompatible(String vin) throws SmartcarException {
     HttpUrl url = HttpUrl.parse(this.urlApi).newBuilder()
         .addPathSegment("compatibility")
         .addQueryParameter("vin", vin)
+        .addQueryParameter("scope", scope)
+        .build();
+
+    Request request = new Request.Builder()
+        .url(url)
+        .header("Authorization", this.basicAuthorization)
+        .addHeader("User-Agent", AuthClient.USER_AGENT)
+        .get()
+        .build();
+
+    return AuthClient.execute(request, Compatibility.class)
+    	  .getData().getCompatible();
+  }
+
+    /**
+   * Determine if a vehicle is compatible with Smartcar given scopes.
+   * Overloads isCompatible function to make scope optional
+   *
+   * @param vin the vin of the vehicle
+   * @param scope the space seperated string of scopes
+   *
+   * @return true if the vehicle is compatible
+   *
+   * @throws SmartcarException when the request is unsuccessful
+   */
+  public boolean isCompatible(String vin, String scope) throws SmartcarException {
```

Comment:
Ugh, I should've mentioned this before. scope should be a list of strings. We will then concat it into a space separated string before adding it onto the query params.